### PR TITLE
About Us CTA added

### PR DESCRIPTION
**Pull request by Divyanshi014**
Did Issue #50 (https://github.com/Codess-Cafe/Codess-Website/issues/50)

BEFORE:
hacktober-1

AFTER:
image

### CODE REVIEW

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Add some spacing between button and text above it . Also, change the color of button a little/apply some styling so that it is not camouflaged with background

**Comment by Divyanshi014** [comment]
sure @VidhiBhatt01!

**Review by VidhiBhatt01 — COMMENTED** [review]
Any updates?

**Comment by Divyanshi014** [comment]
I've completed the coding part, and will surely push the code by evening

**Comment by Divyanshi014** [comment]
Changes in CTA button completed, as per the guidance.
image

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (VidhiBhatt01) [inline .vscode/settings.json:1]: These is not required to be added in commit

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Instead of changing the whole color, just take  a darker shade of gradient/purple. The pinkish color is not at all visually appealing

**Comment by VidhiBhatt01** [comment]
> Changes in CTA button completed, as per the guidance. image

Earlier one was also better than this. Maybe you can add a dark shade border to the button

**Comment by Divyanshi014** [comment]
Sure @VidhiBhatt01 I'm working on these changes.

**Comment by Divyanshi014** [comment]
image
Hello @VidhiBhatt01 ! 
Kindly review the changes.

File: .vscode/settings.json (hunk 1 above)
Comment (VidhiBhatt01) [reply .vscode/settings.json:1]: Remove this

File: style.css [hunk 2]
```diff
@@ -842,7 +842,59 @@ img.logo {
...
   text-align: center;
 }
+/* About Us */
+.btn_aboutUs{
+  display: inline-block;
+  position: relative;
+  min-width: 200px;
+  height: 50px;
+  background: #f16593;
+  border-radius: 60px;
```
Comment (VidhiBhatt01) [inline style.css:852]: lower it a little

File: style.css [hunk 3]
```diff
@@ -1295,7 +1347,7 @@ h1.big-heading {
   /* position:absolute; */
   right: 30%;
   margin-bottom: 50px;
-  margin-top: -50px;
+
```
Comment (VidhiBhatt01) [inline style.css:1350]: why is this removed?

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
"darker shade border" not white. Use a shade of purple

**Comment by Divyanshi014** [comment]
> "darker shade border" not white. Use a shade of purple

image
Is this going good?

File: style.css (hunk 3 above)
Comment (Divyanshi014) [reply style.css:1350]: It was done to make it responsive, I'll add that again.

File: .vscode/settings.json (hunk 1 above)
Comment (VidhiBhatt01) [reply .vscode/settings.json:1]: it is still present

**Review by VidhiBhatt01 — COMMENTED** [review]
live server portion then we are good to go.

**Comment by VidhiBhatt01** [comment]
> > "darker shade border" not white. Use a shade of purple
> 
> image Is this going good?

Better than before, since you are already making a commit for removing live server part, go for a little bit more darker shade of purple. Current one is good too but darker one would look better

**Comment by Divyanshi014** [comment]
Did all the changes, mentioned above. 
AboutUs CTA's CSS is in custom.css
image

**Review by VidhiBhatt01 — APPROVED** [review]
Thanks for contributing!
